PlayniteSounds.cs

```diff
@@ -1165,73 +759,369 @@ public void ImportSounds()
-                    logger.Error(E, "ImportSounds");
-                    PlayniteApi.Dialogs.ShowErrorMessage(E.Message, Constants.AppName);
+                    HandleException(E);
                 }
             }
         }
 
-        public void RemoveSounds()
+        public void OpenSoundManagerFolder()
         {
             try
             {
-                string SoundManagerFilesDataPath = Path.Combine(GetPluginUserDataPath(), "Sound Manager");
                 //just in case user deleted it
-                Directory.CreateDirectory(SoundManagerFilesDataPath);
+                Directory.CreateDirectory(_soundManagerFilesDataPath);
+                Process.Start(_soundManagerFilesDataPath);
+            }
+            catch (Exception E)
+            {
+                HandleException(E);
+            }
+        }
 
-                OpenFileDialog dialog = new OpenFileDialog
-                {
-                    Filter = "ZIP archive|*.zip",
-                    InitialDirectory = SoundManagerFilesDataPath
-                };
-                bool? result = dialog.ShowDialog(PlayniteApi.Dialogs.GetCurrentAppWindow());
-                if (result == true)
+        #endregion
+
+        #endregion
+
+        private bool GetBoolFromYesNoDialog(string caption)
+        {
+            var selection = PlayniteApi.Dialogs.ShowMessage(caption, Resource.SelectOption, MessageBoxButton.YesNo);
+
+            return selection == MessageBoxResult.Yes;
+        }
+
+        #endregion
+
+        #region File Management
+
+        private void InitialCopyAudioFiles()
+        {
+            try
+            {
+                string soundFilesInstallPath = Path.Combine(PluginFolder, SoundDirectory.Sound);
+
+                if (Directory.Exists(soundFilesInstallPath) && !Directory.Exists(_soundFilesDataPath))
                 {
-                    string targetPath = dialog.FileName;
-                    File.Delete(targetPath);
-                    PlayniteApi.Dialogs.ShowMessage(Application.Current.FindResource("LOC_PLAYNITESOUNDS_ManagerDeleteConfirm").ToString() + " " + Path.GetFileNameWithoutExtension(targetPath));
+                    CloseAudioFiles();
+
+                    Directory.CreateDirectory(_soundFilesDataPath);
+                    string[] files = Directory.GetFiles(soundFilesInstallPath);
+                    foreach (string file in files)
+                    {
+                        string destPath = Path.Combine(_soundFilesDataPath, Path.GetFileName(file));
+                        File.Copy(file, destPath, true);
+                    }
                 }
             }
             catch (Exception E)
             {
-                logger.Error(E, "RemoveSounds");
-                PlayniteApi.Dialogs.ShowErrorMessage(E.Message, Constants.AppName);
+                HandleException(E);
             }
         }
 
-        public void OpenSoundManagerFolder()
+        private void SelectMusicFilename()
```

Comment:
I believe I remember @joyrider3774 mentioning implementing multiple select options, so this may be a guaranteed conflict I'll need to resolve.